website/docs/d/is_security_groups.html.markdown

```diff
@@ -0,0 +1,115 @@
+---
+layout: "ibm"
+page_title: "IBM : ibm_is_security_groups"
+description: |-
+  Get information about SecurityGroupCollection
+subcategory: "Virtual Private Cloud API"
+---
+
+# ibm_is_security_groups
+
+Provides a read-only data source for SecurityGroupCollection. You can then reference the fields of the data source in other resources within the same configuration using interpolation syntax.
+
+## Example Usage
+
+```hcl
+data "ibm_is_security_groups" "example" {
+}
+
+OR with Filters:
+
+Filter with VPC name
+data "ibm_is_security_groups" "example" {
+  vpc_name = "<Name of VPC>"
+}
+
+Filter with VPC ID
+data "ibm_is_security_groups" "example" {
+  vpc_id = "<VPC identifier>"
+}
+
+Filter with VPC CRN
```

Comment:
Filter with VPC name should be a comment.. refer the existing resources and add the comments accordingly..
Also please check indentation

Reply:
Made it separate for the filters individually.